### PR TITLE
#2948 - Bond thickness is not changed via Ketcher API

### PR DESCRIPTION
**Pull request by AKZhuk**
Resolves [#2948](https://github.com/epam/ketcher/issues/2948)
## How the feature works?

https://github.com/epam/ketcher/assets/16281184/e7908690-24bc-462e-91be-d6a6902ca920

Fix for:
- Change bond thickness via API
-  Bond thickness for for "save as"  method
-  Bond thickness for for "save as" preview  
-  Bond thickness for "copyToCLipBoard" method

## Check list
- [ ] unit-tests written
- [x] e2e-tests written
- [x] documentation updated
- [x] PR name follows the pattern `#1234 – issue name`
- [x] branch name doesn't contain '#'
- [x] PR is linked with the issue
- [x] base branch (master or release/xx) is correct
- [x] task status changed to "Code review"

### CODE REVIEW

File: packages/ketcher-core/src/application/indigo.ts [hunk 1]
```diff
@@ -200,12 +201,13 @@ export class Indigo {
   ): Promise<string> {
     const outputFormat = options?.outputFormat || 'png';
     const backgroundColor = options?.backgroundColor || '';
-
+    const bondThickness = options?.bondThickness || 1;
```
Comment (Nitvex) [inline packages/ketcher-core/src/application/indigo.ts:204]: I propose to create a constant for default value, which will be used across app :) 
And additional question. Here you have `1` default value, but somewhere below you have `2`. Why? :)

Comment (AKZhuk) [reply packages/ketcher-core/src/application/indigo.ts]: propose to create a constant for default value - _Done_.
Regarding value `2` it is used just for testing

File: packages/ketcher-core/src/application/editor/shared/constants.ts [hunk 2]
```diff
@@ -34,3 +34,5 @@ export const selectionKeys = [
   'simpleObjects',
   'texts',
 ] as const;
+
+export const defaultBondThickness = 1;
```
Comment (KonstantinEpam23) [inline packages/ketcher-core/src/application/editor/shared/constants.ts:38]: Can we please set this to `2` and refer to it in here: https://github.com/epam/ketcher/blob/master/packages/ketcher-react/src/script/ui/data/schema/options-schema.ts#L224

Comment (KonstantinEpam23) [reply packages/ketcher-core/src/application/editor/shared/constants.ts:38]: or probably even create a separate constant for this and refer to it in both places

Comment (AKZhuk) [reply packages/ketcher-core/src/application/editor/shared/constants.ts]: done

**Review by KonstantinEpam23 — APPROVED** [review]

**Pull request merged**